### ISSUE TITLE
Invalid interactive detected on parent nodes

### ISSUE THREAD

**Issue by binhums** [rwjblue/ember-template-lint#80]
Currently the invalid-interactive rule is triggered when a non-interactive element has a child with an interaction - regardless of whether the child is an interactive element. E.g.

```
<li>
  <button {{action 'foo'}}>
</li>
```

I have a fix incoming.
